apps/showcase/src/components/showcase/basic/basic-pres.template.html

```diff
@@ -11,7 +11,7 @@ <h2 class="bg-body-tertiary text-secondary border border-light-subtle border-3 r
             <div aria-hidden="true" class="bg-body-tertiary position-absolute start-0 top-50 border border-light-subtle border-3 border-end-0 border-bottom-0"></div>
           </h2>
         </div>
-        <h5 class="row card-text mt-auto ms-1">Where do you want to go?</h5>
+        <h3 class="row card-text mt-auto ms-1 h5">Where do you want to go?</h3>
```

Comment:
Do we really want to keep headings in the showcase components ?
Because they are not heading related to the app but only for a showcase purpose

Reply:
either way the headings needs to accurately define the structure of the page, which I think is the case here
I can change it if you disagree

Reply:
Yes, I agree it is the good value according to the structure of the page.
Maybe we should use `<p>` or `<div>` instead `<h[x]`> in these showcase components. 
As you want :)